### PR TITLE
CSharpProject Strategy nUnit console output redirection

### PR DESCRIPTION
**Pull request by jicata**
closes https://github.com/SoftUni-Internal/suls-issues/issues/2114

### CODE REVIEW

File: Open Judge System/Workers/OJS.Workers.ExecutionStrategies/CSharpProjectTestsExecutionStrategy.cs [hunk 1]
```diff
@@ -76,12 +96,14 @@ public override ExecutionResult Execute(ExecutionContext executionContext)
 
             // Modify Project file
             var project = new Project(csProjFilePath);
+            var compileDirectory = project.DirectoryPath;
+            this.SetupFixturePath = $"{compileDirectory}\\{SetupFixtureFileName}.cs";
```
Comment (Innos) [inline Open Judge System/Workers/OJS.Workers.ExecutionStrategies/CSharpProjectTestsExecutionStrategy.cs]: Keep consistency with the project, add a runtime constant in the GlobalConstants for the file extension and reuse it.

File: Open Judge System/Workers/OJS.Workers.ExecutionStrategies/CSharpProjectTestsExecutionStrategy.cs [hunk 2]
```diff
@@ -172,6 +200,9 @@ public override ExecutionResult Execute(ExecutionContext executionContext)
 
         private void CorrectProjectReferences(IEnumerable<TestContext> tests, Project project)
         {
+            File.WriteAllText(this.SetupFixturePath, SetupFixtureTemplate);
+            project.AddItem("Compile", $"{SetupFixtureFileName}.cs");
+
             foreach (var testName in this.TestNames)
             {
                 project.AddItem("Compile", $"{testName}.cs");
```
Comment (Innos) [inline Open Judge System/Workers/OJS.Workers.ExecutionStrategies/CSharpProjectTestsExecutionStrategy.cs]: Keep consistency with the project, add a runtime constant in the GlobalConstants for the file extension and reuse it.

File: Open Judge System/Workers/OJS.Workers.ExecutionStrategies/CSharpProjectTestsExecutionStrategy.cs [hunk 3]
```diff
@@ -172,6 +200,9 @@ public override ExecutionResult Execute(ExecutionContext executionContext)
 
         private void CorrectProjectReferences(IEnumerable<TestContext> tests, Project project)
         {
+            File.WriteAllText(this.SetupFixturePath, SetupFixtureTemplate);
+            project.AddItem("Compile", $"{SetupFixtureFileName}.cs");
```
Comment (Innos) [inline Open Judge System/Workers/OJS.Workers.ExecutionStrategies/CSharpProjectTestsExecutionStrategy.cs]: Keep consistency with the project, add a runtime constant in the GlobalConstants for the file extension and reuse it.

File: Open Judge System/Workers/OJS.Workers.ExecutionStrategies/CSharpUnitTestsExecutionStrategy.cs [hunk 4]
```diff
@@ -47,6 +47,8 @@ public override ExecutionResult Execute(ExecutionContext executionContext)
                 CsProjFileSearchPattern);
 
             var project = new Project(csProjFilePath);
+            this.SetupFixturePath = $"{project.DirectoryPath}\\{SetupFixtureFileName}.cs";
```
Comment (Innos) [inline Open Judge System/Workers/OJS.Workers.ExecutionStrategies/CSharpUnitTestsExecutionStrategy.cs]: Keep consistency with the project, add a runtime constant in the GlobalConstants for the file extension and reuse it.

File: Open Judge System/Web/OJS.Web/Controllers/AccountController.cs [hunk 5]
```diff
@@ -3,8 +3,6 @@
     using System;
```
Comment (Innos) [inline Open Judge System/Web/OJS.Web/Controllers/AccountController.cs]: Wrong Account Controller, revert it to the original one.

File: Open Judge System/Workers/OJS.Workers.ExecutionStrategies/CSharpUnitTestsExecutionStrategy.cs [hunk 6]
```diff
@@ -153,14 +165,19 @@ protected override CompileResult Compile(CompilerType compilerType, string compi
 
         private void ExtractTestResult(string receivedOutput, out int passedTests, out int totalTests)
         {
+            // Grabbing the last regex match from the nUnit console
```
Comment (KristianMariyanov) [inline Open Judge System/Workers/OJS.Workers.ExecutionStrategies/CSharpUnitTestsExecutionStrategy.cs]: move this comment as a summary of the method

**Pull request merged**